src/components/menu/menu.tsx

```diff
@@ -275,17 +286,14 @@ export class Menu {
             currentItem = currentItem.parentItem;
         }
 
-        if (breadCrumbItems.length) {
-            breadCrumbItems.push({
-                text: '',
-                icon: {
-                    name: 'home',
-                },
-                type: 'icon-only',
-            } as MenuCrumbItem);
+        if (
+            breadCrumbItems.length ||
+            this.rootItem !== DEFAULT_ROOT_BREADCRUMBS_ITEM
```

Comment:
This means that you'd have to add a custom root item even though you might just want to show the default? Should we make it possible to just choose to show the root item?

Reply:
Yes, but I think you never should just show the default. It has no text and shows an empty tooltip. Showing a generic icon that works well everywhere is difficult, and default text is difficult because it needs to be translated.

So to use breadcrumbs, I think this needs to be provided.

Reply:
I'm a little unsure how to interpret the above. It sounds like you are describing a breaking change, but since the examples are still working as expected, and you haven't had to update any of them in this PR, I assume it's not actually a breaking change? 😅

Reply:
It’s not a breaking change. I’m just complaining a bit about the default home item that’s rendered with an empty tooltip text. It’s better than nothing if you’re showing a sub menu, but I think you’d always want to replace it by providing `rootItem`.

There is a slight inconsistency introduced by always showing the breadcrumbs when `rootItem` is provided, not just after navigating to a submenu. I think this is better, because it means the position of the menu items doesn’t jump up/down when navigating between submenus and root menu.